kinetic/distribution.yaml

```diff
@@ -6141,7 +6141,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/IntelRealSense/librealsense2-release.git
-      version: 2.23.0-1
+      version: 2.25.2-1
     source:
       type: git
       url: https://github.com/IntelRealSense/librealsense.git
```

Comment:
> So, Can you please help me understand why the repository : github.com/IntelRealSense/librealsense is taken for the build?

I think devel jobs Like [K**dev**__librealsense2__ubuntu_xenial_amd64](http://build.ros.org/job/Kdev__librealsense2__ubuntu_xenial_amd64/5/) use the repo from the `source` entry. Did you want to change this to `https://github.com/doronhi/librealsense.git`?